src/poojaRajguru/Test1_FirstNonRepeatedString.java

```diff
@@ -0,0 +1,30 @@
+/*Name : Pooja Rajguru
+Test 1 , 25/12/2021*/
+
+package poojaRajguru;
+
+public class Test1_FirstNonRepeatedString {
+
+	void findChar(String str1){
+		System.out.println("The given string is: " + str1);
+		for (int i = 0; i < str1.length(); i++) {
+			boolean flag = true;
+			for (int j = 0; j < str1.length(); j++) {
+				if (i != j && str1.charAt(i) == str1.charAt(j)) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop